python/ray/services.py

```diff
@@ -3,4 +3,4 @@
 
 # TODO(ekl) deprecate and move this to ray.util
 def get_node_ip_address(address="8.8.8.8:53"):
-    return ray._private.services.get_node_ip_address(address)
+    return ray.util.get_node_ip_address(address)
```

Comment:
Can you please raise a loud deprecation message here?

Reply:
Added, PTAL!